lib/ZumoOta/src/App.h

```diff
@@ -82,9 +81,15 @@ class App
      */
     void loop();
 
-/******************************************************************************
- * Functions
- *****************************************************************************/
+    /**
+     * Start the Application.
+     */
+    void start();
+
+private:
+    FileManager fileManager{};

```

Comment:
Prefix member variables with "m_". I saw that Gabryel follows this convention, so it should be consistent.

Reply:
I will perform it with the next commit.